src/Compilers/CSharp/Portable/Symbols/Attributes/WellKnownAttributeData/PropertyWellKnownAttributeData.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Information decoded from well-known custom attributes applied on a property.
+    /// </summary>
+    internal sealed class PropertyWellKnownAttributeData : CommonPropertyWellKnownAttributeData
+    {
+        // TODO2 remove this type
```

Comment:
>// TODO2 remove this type [](start = 8, length = 25)

Please restore handling of these attributes as well known attributes

Reply:
You mean restoring what I had in previous iteration?
I don't think I see a benefit. We need to inspect the attribute data to find the correct one to copy anyways.

---
In reply to: [295560074](https://github.com/dotnet/roslyn/pull/36444#discussion_r295560074) [](ancestors = 295560074)

Reply:
> I don't think I see a benefit. We need to inspect the attribute data to find the correct one to copy anyways.

The benefit is performance. Why do you think we have all that infrastructure when we could always do what you did to see if there is a well-known attribute? 

---
In reply to: [295573079](https://github.com/dotnet/roslyn/pull/36444#discussion_r295573079) [](ancestors = 295573079,295560074)

Reply:
How often do we copy the attributes and how often do we check for annotations during flow analysis?

---
In reply to: [295741201](https://github.com/dotnet/roslyn/pull/36444#discussion_r295741201) [](ancestors = 295741201,295573079,295560074)